plugins/vmware_vcenter.yaml

```diff
@@ -0,0 +1,30 @@
+# Plugin Info
+version: 0.0.1
+title: VMware vCenter
+description: Log parser for VMware vCenter
+parameters:
+  listen_address:
+    label: Listen Address
+    description: A syslog address of the form `<ip>:<port>`
+    type: string
+    default: ":5140"
+
+# Set Defaults
+{{$listen_address := default ":5140" .listen_address}}
+
+# Pipeline Template
+pipeline:
+  - id: vcenter_input
+    type: tcp_input
+    listen_address: {{ $listen_address }}
+    labels:
+      log_type: vmware_vcenter 
+    output: vcenter_parser
+
+  - id: vcenter_parser
+    type: regex_parser
+    regex: '^[<\d+>]*\s*(?P<timestamp>[^ ]+) (?P<hostname>[^ ]+) (?P<service>[^ ]+) (?P<message>.*)'
```

Comment:
syslog?

Reply:
This is the same problem that I ran into with cisco meraki. The sample logs they saved, along with the fluentd configuration they used, don't allow me to safely make that conclusion. I don't know if these systems just use raw UDP/TCP, or if they use syslog and the people who made the log bundles failed to understand how that works.